src/Entities/Entity.h

```diff
@@ -289,14 +289,16 @@ class cEntity
 	If this returns false, you must stop using the cEntity pointer you have. */
 	bool IsTicking(void) const;
 
+	// tolua_end
 	/** Destroys the entity, schedules it for memory freeing and broadcasts the DestroyEntity packet */
 	virtual void Destroy();
 
 	OBSOLETE void Destroy(bool a_ShouldBroadcast)
 	{
-		LOGWARNING("cEntity:Destory(bool) is deprecated, use cEntity:Destroy() instead.");
+		LOGWARNING("cEntity:Destroy(bool) is deprecated, use cEntity:Destroy() instead.");
 		Destroy();
 	}
```

Comment:
I believe this only existed for the benefit of the lua bindings. Should be safe to delete.

Reply:
The warning or the whole function?

Reply:
whole function